lib/node_modules/@stdlib/assert/is-cube-number/benchmark/benchmark.js

```diff
@@ -135,15 +91,12 @@ bench( pkg+'::primitives:isPrimitive', function benchmark( b ) {
 	b.end();
 });
 
-bench( pkg+'::objects:isPrimitive', function benchmark( b ) {
+bench( pkg+'::objects,true', function benchmark( b ) {
 	var values;
 	var bool;
 	var i;
 
 	values = [
-		[],
-		{},
-		function noop() {},
 		new Number( 8.0 ),
 		new Number( 0.5 ),
 		new Number( NaN ),
```

Comment:
You'll have to put those values that results in `true`

Reply:
should I change those values for new ones then?

Reply:
Yes

Reply:
ok